packages/Sortable/README.md

```diff
@@ -0,0 +1 @@
+Sortable items are rad.
```

Comment:
TLDR

Reply:
can we add a simple example about how to use the component, this is our basic documentation that will appear on the npm page